### ISSUE TITLE
Support for KMS server-side encryption?

### ISSUE THREAD

**Issue by zezke** [markitx/dynamo-backup-to-s3#39]
I've made some changes to support KMS encryption. You can just provide a _kmsKeyId_ and it'll set everything up.

Is this interesting enough to warrant a PR?

**Comment by kileynaromw** [comment]
Hi zezke! Your request appears to be a bit different at first glance, but you may be interested in the PR I opened back on May 20th:

https://github.com/markitx/dynamo-backup-to-s3/pull/41

Having this PR merged would allow me as a caller to say something like:
```
backup = new DynamoBackup({
    ...,
    ServerSideEncryption = 'AES256'
});
```

It looks like your request KMS encryption is a bit different because my ServerSideEncryption change above uses a default key on the server, but I thought I'd mention it.

**Comment by dylanlingelbach** [comment]
@zezke yes, this warrants a PR. I lost my write permissions to this repo so trying to figure out if I need to move it. It may take me a while to merge a PR.
